api/server/router/network/network_routes.go

```diff
@@ -236,14 +245,133 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 			return err
 		}
 		nw = &types.NetworkCreateResponse{
-			ID:      id,
-			Warning: warning,
+			ID: id,
 		}
 	}
 
+	nw.Warning = strings.Join(warnings, ". ") + "."
 	return httputils.WriteJSON(w, http.StatusCreated, nw)
 }
 
+type ipFamily int
+
+const (
+	ip4 ipFamily = 4
+	ip6 ipFamily = 6
+)
```

Comment:
Looks like we're always assigning these values internally, so we may as well save the pain of having to format this into a human-readable string;

```go
type ipFamily string

const (
	ip4 ipFamily = "IPv4"
	ip6 ipFamily = "IPv6"
)
```